buildkite/scripts/tests/archive-node-unit-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [[ ! -f dune-project ]]; then
+    echo "Error: This script must be run from the root of the repository (where 'dune-project' exists)."
+    exit 1
+fi
+
+user="$1"
+password="$2"
+db="$3"
+command_key="$4"
+
+if [[ -z "$user" || -z "$password" || -z "$db" || -z "$command_key" ]]; then
+    echo "Usage: $0 <user> <password> <db> <command_key>"
+    exit 1
+fi
+
+eval $(opam config env) 
```

Comment:
This line used to panic with OPAM not found, but once you move it in a scripts it's fine...

I'm suspecting the original line is running outside of the docker image? I'm not familiar with dhall's syntax, though. 

@SanabriaRusso WDYT?